lib/__test__/rule-list.test.js

```diff
@@ -380,6 +386,31 @@ describe('rules', () => {
         ['x^-a / x^-b', 'x^(-a - -b)'],
     ])
 
+    suite('multiplying coefficients', rules.MULTIPLY_COEFFICIENTS, [
+        ['x^2 * y^2', '(1 * 1) (x^2 * y^2)'],
```

Comment:
This feels like a really result.  We end up with a more complicated expression that actually contains the original expression.  Maybe check with Gen to see if we actually want this.